src/meta/metainfo.go

```diff
@@ -131,6 +131,32 @@ func (i *info) FindConstants(substr string) (res []string) {
 	return res
 }
 
+func (i *info) InitKphpStubs() {
```

Comment:
Can this function be moved to `linter` package so it can check the `kphp` flag?
Or we can pass it to the Init stubs function to avoid redundant meta overrides for non-kphp mode.